src/fides/api/ops/api/deps.py

```diff
@@ -5,13 +5,12 @@
 
 from fides.api.ops.common_exceptions import FunctionalityNotConfigured
 from fides.api.ops.util.cache import get_cache as get_redis_connection
-from fides.core.config import FidesConfig
+from fides.core.config import CONFIG, FidesConfig
```

Comment:
As far as I can tell we're not actually using `CONFIG` in this file. Looks like `get_app_config()` is what's used.